lightning_talks.yml

```diff
@@ -16,3 +16,10 @@ events:
     description: |
       A short overview of how we made automatized pentesting in SCS actually.
     location: "https://conf.scs.koeln:8443/SCS-Tech"
+  - summary: "Overview of the Moin Cluster"
+    begin: 2024-06-27 15:45:00         # uses default timezone above
+    duration:
+      minutes: 15
+    description: |
+      Let's discover the Moin Cluster.
```

Comment:
A little information in advance about what a "Moin Cluster" is would be good IMO.

Reply:
Thats whats the talk is about, but I can add some text later